packages/http-specs/spec-summary.md

```diff
@@ -1661,6 +1661,160 @@ Content-Type: application/octet-stream
 --abcde12345--
 ```
 
+### Payload_Pageable_ServerDrivenPagination_ContinuationToken_reqHeaderResBody
+
+- Endpoint: `get /payload/pageable/server-driven-pagination/continuationtoken/request-header-response-body`
+
+Test case for using continuation token as pagination. Continuation token is passed in the request header and response body.
+
+Two requests need to be tested.
+
+1. Initial request:
+   Expected route: /payload/pageable/server-driven-pagination/continuationtoken/request-header-response-body
+
+Expected response body:
+
+```json
+{
+  "pets": [
+    { "id": "1", "name": "dog" },
+    { "id": "2", "name": "cat" }
+  ],
+  "nextToken": "12345"
+}
+```
+
+2. Next page request:
+   Expected route: /payload/pageable/server-driven-pagination/continuationtoken/request-header-response-body
+
+Expected header:
+token=12345
+
+Expected response body:
+
+```json
+{
+  "pets": [
+    { "id": "3", "name": "bird" },
+    { "id": "4", "name": "fish" }
+  ]
+}
+```
+
+### Payload_Pageable_ServerDrivenPagination_ContinuationToken_reqHeaderResHeader
+
+- Endpoint: `get /payload/pageable/server-driven-pagination/continuationtoken/request-header-response-header`
+
+Test case for using continuation token as pagination. Continuation token is passed in the request header and response header.
+
+Two requests need to be tested.
+
+1. Initial request:
+   Expected route: /payload/pageable/server-driven-pagination/continuationtoken/request-header-response-header
+   Expected response body:
+
+```json
+{
+  "pets": [
+    { "id": "1", "name": "dog" },
+    { "id": "2", "name": "cat" }
+  ]
+}
+```
+
+Expected response header:
+next-token=12345
+
+2. Next page request:
+   Expected route: /payload/pageable/server-driven-pagination/continuationtoken/request-header-response-header?token=12345
```

Comment:
Why is token included as request header and query parameter?

Reply:
@JoshLove-msft Here is detailed context: https://github.com/Azure/typespec-azure/issues/2141#issue-2814381655

Reply:
My question is why is it included in both the query param and the request header **in the same** scenario.

Reply:
@msyyc Probably just a wrong doc here, as it is request-header case?